File: parte001/ex004_aritmetica.java
//  Escriba un algoritmo que, dados dos valores A y B, encuentre:  
// A - B 
// A + B 
// A * B 
// A / B
// A % B

import java.util.Scanner;

public class ex004_aritmetica {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a, b;
        
        System.out.print("Ingrese el valor de A: ");
        a = sc.nextInt();
        
        System.out.print("Ingrese el valor de B: ");
        b = sc.nextInt();
        
        System.out.println("A - B = " + (a - b));
        System.out.println("A + B = " + (a + b));
        System.out.println("A * B = " + (a * b));
        System.out.println("A / B = " + (a / b));
        System.out.println("A % B = " + (a % b));
    }
}

File: parte001/ex004_aritmetica.cpp
//  Escriba un algoritmo que, dados dos valores A y B, encuentre:  
// A - B 
// A + B 
// A * B 
// A / B
// A % B

#include <iostream>

using namespace std;

int main()
{
    int a, b;
    cout << "Ingrese dos valores enteros: ";
    cin >> a >> b;
    
    cout << "A - B = " << a - b << endl;
    cout << "A + B = " << a + b << endl;
    cout << "A * B = " << a * b << endl;
    cout << "A / B = " << a / b << endl;
    cout << "A % B = " << a % b << endl;
    
    return 0;
}
